apps/services/license-api/src/test/setup.ts

```diff
@@ -0,0 +1,20 @@
+import { Type } from '@nestjs/common'
+import { TestApp, testServer, TestServerOptions } from '@island.is/testing/nest'
+
+export let app: TestApp
+
+export const setup = (
+  module: Type<any>,
```

Comment:
Can we use `unknown`? If not `eslint-disable`

Reply:
Tests are on the docket for the future, for now I'll disable the warning